### ISSUE TITLE
The problem of deleted file is still in problems panel.

### ISSUE THREAD

**Issue by zuoez02** [vuejs/vetur#1147]
- [x] I have searched through existing issues
- [x] I have read through [docs](https://vuejs.github.io/vetur)
- [x] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: macOS
- Vetur version: 0.16.2
- VS Code version: 1.33.0-insider

## Problem

The problems in deleted files are still exist in PROBLEMS panel.

Actually, it is also a bug on both code and code-insider. I just use code-insider as a clean version without other extensions.

## Reproducible Case
1. Generate a new vue project with vue-cli 3 with Typescript.
2. Write a .vue file with problem.
```vue
<template>
<div>{{ hello }}</div>
</template>
<script lang="ts">
import Vue from 'vue';
import Component from 'vue-class-component';

@Component
export default class File extends Vue {
    public hello: string = 123;
}
</script>
```
3. It said `Type '123' is not assignable to type 'string'.` in PROBLEM panel.
4. Delete the file
5. It is still there. Click the file and vscode says `File not found`
6. Reload the Code and problem is not shown anymore.

**Comment by octref** [comment]
Should be fixed by #1182.

**Comment by livevsonline** [comment]
Hmm i just had this in a freshly installed project, vue 3 with prettier
